apps/api/src/app/events/e2e/bulk-trigger.e2e.ts

```diff
@@ -212,73 +211,67 @@ describe('Trigger bulk events - /v1/events/trigger/bulk (POST)', function () {
-        },
-        {
-          headers: {
-            authorization: `ApiKey ${session.apiKey}`,
-          },
-        }
-      );
+      await novuClient.triggerBulk({
+        events: Array.from({ length: 101 }, () => event),
+      });
     } catch (e) {
       error = e;
     }
+    const { error: sdkError, parsedBody } = handleSdkError(error);
 
-    expect(error).to.be.ok;
-    expect(error.response.status).to.equal(400);
-    expect(error.response.data.message[0]).to.equal('events must contain no more than 100 elements');
+    expect(sdkError.statusCode).to.equal(400);
+    expect(parsedBody.statusCode).to.equal(400);
+    expect(parsedBody.message[0]).to.equal('events must contain no more than 100 elements');
   });
 
   it('should handle bulk if one of the events returns errors', async function () {
-    const response = await axiosInstance.post(
-      `${session.serverUrl}${BULK_API_ENDPOINT}`,
-      {
-        events: [
-          {
-            transactionId: '1111',
-            name: 'non-existing-trigger',
-            to: subscriber.subscriberId,
-            payload: {
-              firstName: 'Testing of User Name',
-              urlVariable: '/test/url/path',
-            },
+    const bulkTriggerResponse = await triggerBulk(novuCore, {
+      events: [
+        {
+          transactionId: '1111',
+          name: 'non-existing-trigger',
+          to: [subscriber.subscriberId],
+          payload: {
+            firstName: 'Testing of User Name',
+            urlVariable: '/test/url/path',
           },
-          {
-            transactionId: '2222',
-            name: template.triggers[0].identifier,
-            to: subscriber.subscriberId,
-            payload: {
-              firstName: 'Testing of User Name',
-              urlVariable: '/test/url/path',
-            },
+        },
+        {
+          transactionId: '2222',
+          name: template.triggers[0].identifier,
+          to: [subscriber.subscriberId],
+          payload: {
+            firstName: 'Testing of User Name',
+            urlVariable: '/test/url/path',
           },
-          {
-            transactionId: '1111',
-            payload: {
-              firstName: 'Testing of User Name',
-              urlVariable: '/test/url/path',
-            },
+        },
+        {
+          transactionId: '1111',
+          payload: {
+            firstName: 'Testing of User Name',
+            name: '',
           },
-        ],
-      },
-      {
-        headers: {
-          authorization: `ApiKey ${session.apiKey}`,
+          name: '',
+          to: [],
         },
-      }
-    );
+      ],
+    });
+    if (!bulkTriggerResponse.ok) {
+      throw new Error(`failed to bulk trigger:${JSON.stringify(bulkTriggerResponse.error)}`);
+    }
 
-    const { data: body } = response;
-    expect(body.data).to.be.ok;
-    expect(body.data.length).to.equal(3);
+    const dtoList = bulkTriggerResponse.value.result;
```

Comment:
This `.value.result` duplication is subpar DX. Can we remove the `.value` part?

Reply:
the value belongs to the reilroad programming, the result belongs to the response always being wrapped, we get the body and the headers where the body is in the results, what do you want to do?

Reply:
Ideally, it should be `bulkTriggerResponse.result` and `bulkTriggerResponse.error`, right? Do we have leverage to change that on the speakeasy generated code?

Reply:
This was resolved offline. We will chat with speakeasy to get flatter responses.